source/Halibut.Tests/FriendlyHtmlPageTests.cs

```diff
@@ -152,5 +152,16 @@ static async Task<string> DownloadStringIgnoringCertificateValidation(string uri
                 }
             }
         }
+
+        HalibutRuntime GetHalibutRuntime(IServiceFactory serviceFactory, SyncOrAsync syncOrAsync)
```

Comment:
nit: `IServiceFactory serviceFactory` is never customised, it's always `GetDelegateServiceFactory()` s this param could be removed to reduce plumbing code.

Reply:
I flip-flopped on whether to do this or not, ultimately deciding to leave it as-is. I will update and remove the unecessary plumbing.